<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectionDialog</class>
 <widget class="QWidget" name="connectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <property name="verticalSpacing">
      <number>30</number>
     </property>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioSimulator">
       <property name="text">
        <string>Simulator</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Send back responses manually, eg with recorded xml messages</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="simulatorDirectory">
       <property name="placeholderText">
        <string>Data files directory</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="radioCodec">
       <property name="text">
        <string>Real codec</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Forward messages and responses from real codec. 
Record xml messages to file</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="codecDetails">
       <item>
        <widget class="QLineEdit" name="codecIp">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Codec IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="codecUsername">
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QPushButton" name="connect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
